staging/src/k8s.io/client-go/openapi/openapitest/fakeclient.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openapitest
+
+import (
+	"k8s.io/client-go/openapi"
+)
+
+// FakeClient implements openapi.Client interface, with hard-coded
+// return values, including the possibility to force errors.
+type FakeClient struct {
```

Comment:
Initially wondered if we should write tests for that, but I would advise against it, there's is basically no logic here to test.